src/modules/hyprland/workspaces.cpp

```diff
@@ -339,6 +339,7 @@ void Workspaces::on_workspace_renamed(std::string const &payload) {
       break;
     }
   }
+  sort_workspaces();
```

Comment:
So does this fix the issue? Did you test it?

Reply:
Heho,

worked on my machine. :D